api/prometheus/v1/api.go

```diff
@@ -281,6 +284,16 @@ type ConfigResult struct {
 // FlagsResult contains the result from querying the flag endpoint.
 type FlagsResult map[string]string
 
+// BuildinfoResult contains the results from queyring the buildinfo endpoint
+type BuildinfoResult struct {
+	Version   string `json:"version"`
+	Revision  string `json:"revision"`
+	Branch    string `json:"branch"`
+	BuildUser string `json:"buildUser"`
+	BuildDate string `json:"buildDate"`
```

Comment:
This should be type `time.Time` instead.

Reply:
At first, I've though the same :smile: it is string actually.

https://github.com/prometheus/prometheus/blob/main/web/api/v1/api.go#L109

Reply:
Interesting, wonder why the difference. 🤔 Thanks!

Reply:
It's because it's a string in https://github.com/prometheus/common/blob/main/version/info.go , and I guess it's a string there because it's populated from command line flags.

Reply:
> and I guess it's a string there because it's populated from command line flags.

That makes sense, thanks!